### DISCUSSION TITLE
toefl-exam-experience/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#4] [Announcements]
# https://aprd.ir/toefl-exam-experience/

تجربیات من در آزمون تافل

https://aprd.ir/toefl-exam-experience/

**Comment by pourmand1376** [comment]
این کامنت از سایت Graphcomment - که قبلا استفاده می‌کردم - استخراج شده است. 

**نویسنده کامنت:** «فاطمه»
**متن کامنت:**
ممنون. مفید بود.

**Comment by pourmand1376** [comment]
**نویسنده کامنت:    [فاطمه - ویرگول](https://virgool.io/@fatemeroshani7899)
**متن کامنت:**
سلام آقای پورمند  
از اطلاعات خوبتون خیلی ممنونم  
یک سوال داشتم ؛ اگه موقع امتحان هنوز ۷۰ واحد رو نگذرونده باشیم( یعنی تو اون ترم در حال پاس کردن اون ۷۰ واحد باشیم، مثلا من این ترم رو که بگذرونم میشه ۶۸ واحد ولی موقع برگزاری امتحان تعداد واحد های تو سبد درسیم بیشتر از ۷۰ واحد میشه) ولی دانشجوی سال سوم باشیم، میتونیم تو المپیاد شرکت کنیم؟  
این مشکل رو به این خاطر دارم که تو ترم اول ۱۲ واحد پیش خوردم که متاسفانه جزو واحد های گذرونده حساب نمیشن

----
این کامنت از سایت ویرگول که قبلا استفاده می‌کردم استخراج شده است.

**Reply by pourmand1376** [reply]
قطعا اگر رزومه‌ام در حدی باشه که با مدرک کارشناسی ام بتونم اپلای کنم تو ایران ارشد نمی‌خونم. اصلا پوینتی نداره ارشد خوندن تو ایران ...
اگر کسی می‌خواد دکترا بخونه که دکترا مستقیم خیلی جاها می‌تونه اپلای کنه و اگرم می‌خواد ارشد بخونه و وارد بازار کار بشه بازم اپلای کردن با مدرک کارشناسی منطقی تره.

**Comment by pourmand1376** [comment]
**نویسنده کامنت:   [🍁 Yogi 🍁 - ویرگول](https://virgool.io/@YogiHCl)
**متن کامنت:**
دمت گرم که این تجربه هارو باما به اشتراک میزاری

----
این کامنت از سایت ویرگول که قبلا استفاده می‌کردم استخراج شده است.
